WordPress/gutenberg
# Block Library: Add Comments Pagination Numbers block

## Description

Add a block that renders links for different page numbers inside the Comments Pagination block. It works similar to the Query Pagination Numbers block.

Will close #35010

## How has this been tested?
1. Inside the Single Post template, add a **Comments Query Loop** block.
2. Add a **Post Comments** block as well, to check both blocks behave similarly.
3. Add a **Comments Pagination** block inside the **Comments Query Loop** block.
3. Check that the editor shows a placeholder with numbers.
4. In the frontend, navigate to a post with several comments.
5. Check that the default comments page is the correct one.
6. Click the numbers to navigate through different comment pages.
7. Click the next/previous links inside the **Post Comments** block and check that the **Comments Pagination Numbers** block highlights the correct page number.


## Types of changes
- New feature

## Checklist:
